osler/assets/inventory/drug-list/DrugListTable.js

```diff
@@ -0,0 +1,193 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import TableManager from '../../core/all-patients/TableManager';
+import CSRFToken from '../../util/CSRFToken';
+import compare from '../../util/compare';
+import simpleComparator from '../../util/simpleComparator';
+import Container from 'react-bootstrap/Container';
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import Form from 'react-bootstrap/Form';
+import globalFilter from './globalFilter';
+import { DateTime } from 'luxon';
+
+
+function DrugListTable(props) {
+
+  const [state, setState] = useState({ 'show': false, 'drug': {} });
+
+  const handleClose = () => {
+    setState({ 'show': false, 'drug': {} });
+  }
+  const handleShow = (e, row) => {
+    setState({ 'show': true, 'drug': row });
+  }
+
+  const nameComparator = simpleComparator('name');
+  const categoryComparator = simpleComparator('category');
+  const expirationDateComparator = simpleComparator('expiration_date');
+  const stockComparator = simpleComparator('stock');
+
+  const manufacturerComparator = React.useMemo(() => (rowA,rowB,columnId,desc) => {
+    let cmp = compare(rowA.original.unit,rowB.original.unit);
+    if (cmp == 0) {
+      cmp = compare(rowA.original.manufacturer.toLowerCase(), rowB.original.manufacturer.toLowerCase());
+    }
+    return cmp;
+  });
```

Comment:
There shouldn't be any comparison of units. The simple lowerCase() comparator should work fine here